pytest.ini

```diff
@@ -2,3 +2,4 @@
 log_cli_level = DEBUG
 log_level = DEBUG
 asyncio_mode=strict
+norecursedirs = .* src/bandersnatch_docker_compose
```

Comment:
What does .* do here? Do we want that? Your goal is to just ignore your compose files right?

Reply:
it will work, but the aim here is to avoid using `/` as it is a Linux directory format. So adding a `\` will not affect the result.

I will troubleshoot where exactly is the place with the issue in a separate pull request.
- Till then, i suggest we keep only the `norecursedirs = src\bandersnatch_docker_compose` to the `pytest.ini` so the test will ignore the compose directory.